Packs/BluelivThreatCompass/ReleaseNotes/1_0_12.md

```diff
@@ -0,0 +1,7 @@
+#### Integrations
+
+##### Blueliv ThreatCompass
+
+-   **Fetch incidents** refactorization, the API use page approach instead offset
+-   Fix default order parameter value in search endpoint
+-   Updated the Docker image to: _demisto/python3:3.10.13.84405_.
```

Comment:
```suggestion
-   Fixed an issue where **Fetch incidents** failed due to usage of deprecated query params.
-   Note: **Fetch incidents** will now return results in ascending order rather than descending.
-   Updated the Docker image to: *demisto/python3:3.10.13.84405*.
```